coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 98.52
+    "covered_percent": 98.44
```

Comment:
That is an awesome percentage of code coverage - I didn't see a badge for this on your README.md?